packages/opentelemetry-metrics/test/Meter.test.ts

```diff
@@ -714,6 +733,149 @@ describe('Meter', () => {
     });
   });
 
+  describe('#UpDownSumObserverMetric', () => {
+    it('should create an UpDownSum observer', () => {
+      const upDownSumObserver = meter.createUpDownSumObserver(
+        'name'
+      ) as UpDownSumObserverMetric;
+      assert.ok(upDownSumObserver instanceof Metric);
+    });
+
+    it('should return noop observer when name is invalid', () => {
+      const spy = sinon.stub(meter['_logger'], 'warn');
+      const upDownSumObserver = meter.createUpDownSumObserver('na me');
+      assert.ok(upDownSumObserver === api.NOOP_UP_DOWN_SUM_OBSERVER_METRIC);
+      const args = spy.args[0];
+      assert.ok(
+        args[0],
+        'Invalid metric name na me. Defaulting to noop metric implementation.'
+      );
+    });
+
+    it('should create observer with options', () => {
+      const upDownSumObserver = meter.createUpDownSumObserver('name', {
+        description: 'desc',
+        unit: '1',
+        disabled: false,
+      }) as ValueObserverMetric;
```

Comment:
Is this type casting necessary? Should this be type `UpDownSumObserverMetric`?

Reply:
casting is needed but it should be UpDownSumObserverMetric